asmcli/lib/validate.sh

```diff
@@ -330,7 +330,11 @@ exit_if_cluster_unlabeled() {
   local LABELS; LABELS="$(get_cluster_labels)";
   local REQUIRED; REQUIRED="$(mesh_id_label)";
   local NOTFOUND; NOTFOUND="$(find_missing_strings "${REQUIRED}" "${LABELS}")"
-
+  local UI_DISABLED; UI_DISABLED="$(context_get-option "UI_DISABLED")"
+  if [[ -n "${UI_DISABLED}" ]]; then
+    warn "UI Disabled by command line option; skipping cluster label check."
```

Comment:
1) I would do info instead of warn since ostensibly the user's intent is to disable it.

2) language nit: I think "Skipping cluster label check because UI not enabled." flows better

Reply:
Done.